Source/ACE.Entity/AcePortalObject.cs

```diff
@@ -4,22 +4,19 @@
 namespace ACE.Entity
 {
     [DbTable("vw_ace_portal_object")]
-    [DbGetList("vw_ace_portal_object", 14, "baseAceObjectId")]
+    [DbGetList("vw_ace_portal_object", 14, "AceObjectId")]
     public class AcePortalObject : AceObject
     {
```

Comment:
it's entirely possible this object just goes away if we expose positions in AceObject itself.  We just have the portal code look for the proper position type.